### ISSUE TITLE
Nextcloud docker Internal Server Error with MariaDB 10.6

### ISSUE THREAD

**Issue by freekvh** [nextcloud/docker#1540]
Last week I updated all my Docker images, and Nextcloud updated to 22.0.0. At the same time MariaBD Updated from 10.5 to 10.6. And Nextcloud broke:

```
Internal Server Error

The server was unable to complete your request.

If this happens again, please send the technical details below to the server administrator.

More details can be found in the server log.
Technical details

    Remote Address: x.x.x.x
    Request ID: x

```

From the logs (let me know if you need more):

```
OC\DB\Exceptions\DbalException: An exception occurred while executing a query: SQLSTATE[HY000]: General error: 4047 InnoDB refuses to write tables with ROW_FORMAT=COMPRESSED or KEY_BLOCK_SIZE.
```

Forcing MariaDB back to 10.5 fixed it for me. This may be related: https://github.com/nextcloud/docker/commit/d7f1fa7b0d662b6d33164546fc3b4e83b9bef910

**Comment by deanpcmad** [comment]
This issue might help #1536

**Comment by ThomasMarangoni** [comment]
Adding `command:  --skip-innodb-read-only-compressed` as mentioned in #1536 to my docker-compose file for db section solved the issue.

**Comment by NinjaEde** [comment]
@ThomasMarangoni works great. 
Thanks mate, you saved my day

**Comment by aanno** [comment]
I've got 2 remarks on this:

1. According to https://mariadb.com/kb/en/upgrading-from-mariadb-105-to-mariadb-106/#innodb-compressed-row-format mariadb wants to deprecate the writeable compressed format and get away from it. Hence sticking to 10.5 seems not to be a long-term solution.
2. I was able to circumvent the problem by removing the 4-byte unicode support. See https://dba.stackexchange.com/questions/256427/unable-to-create-tables-with-row-format-compressed/295643#295643 for details. However, I'm not aware about the connection between nextcloud, writeable compressed format and 4-byte unicode. Does anyone know some technical details?
